docs/ext/commands/api.rst

```diff
@@ -250,6 +250,36 @@ are custom to the command extension module.
     :param ctx: The invocation context.
     :type ctx: :class:`.Context`
 
+.. function:: disnake.ext.commands.on_slash_command_completion(inter)
+
+    An event that is called when a slash command has completed its invocation.
+
+    This event is called only if the slash command succeeded, i.e. all checks have
+    passed and the user input it correctly.
```

Comment:
```suggestion
    passed and the command handler ran successfully.
```
I feel like this works better, since correct user input can still lead to exceptions, so that `on_*_command_completion` would not be called

Reply:
> since correct user input can still lead to exceptions

For example?

Reply:
```py
@bot.slash_command()
async def cmd(self, inter, num: int) -> None:
    await inter.send(42 / num)
```
Raises a `ZeroDivisionError` if the user passes `0`. I'd argue `0` is "correct user input", but the completion handler wouldn't be called in that case

Reply:
Ohh I see, thank you, I'm going to commit the changes later.

Reply:
Imagine not using `Param(gt=0)` 🙃